test/NuGet.Core.Tests/NuGet.Protocol.Tests/HttpSourceResourceProviderTests.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using NuGet.Configuration;
+using NuGet.Protocol.Core.Types;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.Protocol.Tests
+{
+    public class HttpSourceResourceProviderTests
+    {
+#if IS_DESKTOP
+        [PlatformFact(Platform.Windows)]
+        public async Task WhenMaxHttpRequestsPerSourceIsNotConfiguredThenItsValueIsSetToDefault_SuccessAsync()
+        {
+            // Arrange
+            var packageSource = new PackageSource("https://contoso.com/v3/index.json");
```

Comment:
```suggestion
            var packageSource = new PackageSource("https://contoso.test/v3/index.json");
```

We should never use potentially real URLs, since we don't want tests to accidentally send HTTP requests to real servers.